### PR TITLE
Create eip155-1455.json

### PR DESCRIPTION
**Pull request by ctextoken**

### CODE REVIEW

**Comment by ctextoken** [comment]
please add ctex scan blockchain to chainlist

**Comment by ctextoken** [comment]
please add ctex scan blockchain to chainlist

File: _data/chains/eip155-1455.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "decimals": 18
+  },
+  "infoURL": "https://ctextoken.io",
+  "shortName": "CTEX",
+  "chainId": 1455,
+  "networkId": 1455,
+  "explorers": [
+    {
+      "name": "Ctex Scan Explorer",
+      "url": "https://ctexscan.com/",
```
Comment (ligi) [inline _data/chains/eip155-1455.json:19]:
```suggestion
      "url": "https://ctexscan.com",
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Review by ligi — APPROVED** [review]

**Comment by ctextoken** [comment]
hello @ligi  when it will show on https://chainlist.org/ and also i have created pull request for rpc adding kindly approve that also https://github.com/DefiLlama/chainlist/pull/301

**Comment by ligi** [comment]
I do not care for Defi and/or Lamas

**Comment by ctextoken** [comment]
ok sorry for that , please add ctex chain to chainlist.org website

**Comment by ctextoken** [comment]
also where else we can add ctex chain

**Comment by ctextoken** [comment]
hello @ligi  please add it on https://chainlist.org/ and https://chainlist.wtf/

**Pull request merged**